rules/rules-overridden-azure/os/windows/tests/os-specific-target-azure-appservice.windup.test.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ruletest id="os-specific-test" xmlns="http://windup.jboss.org/schema/jboss-ruleset" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://windup.jboss.org/schema/jboss-ruleset http://windup.jboss.org/schema/jboss-ruleset/windup-jboss-ruleset.xsd">
+    <testDataPath>../../../../rules-reviewed/os/windows/tests/data</testDataPath>
+    <rulePath>../../../../rules-reviewed/os/windows/tests/os-specific.windup.test.xml</rulePath>
```

Comment:
```suggestion
    <rulePath>../../../../rules-reviewed/os/windows/os-specific.windup.xml</rulePath>
```

Reply:
Thanks. I've fixed it in both test cases.